plugins/modules/mount.py

```diff
@@ -93,7 +93,10 @@
   boot:
     description:
       - Determines if the filesystem should be mounted on boot.
-      - Only applies to Solaris systems.
+      - Only applies to Solaris and Linux systems.
+      - For Solaris systems, C(yes) will set C(yes) as the value of mount at boot
```

Comment:
```suggestion
      - For Solaris systems, C(yes) will set C(yes) as the value of mount at boot.
```

Reply:
I'll fix like below(add "." to the end of next line:
```
      - For Solaris systems, C(true) will set C(yes) as the value of mount at boot
        in I(/etc/vfstab).
```